src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -147,31 +147,27 @@ public Map<String, List<MItemBlock>> getCreativeTabMap() {
 				if (!tabItems.isEmpty()) {
 					for (TabEntry tabEntry : tabElement.getCreativeTabs()) {
 						String tab = tabEntry.getUnmappedValue();
-						if (!tabMap.containsKey(tab))
-							tabMap.put(tab, new ArrayList<>());
 
 						// If tab does not have custom order, add items to the end of the list
 						if (workspace.getCreativeTabsOrder().get(tab) == null)
-							tabMap.get(tab).addAll(tabItems);
+							tabMap.computeIfAbsent(tab, e -> new ArrayList<>()).addAll(tabItems);

```

Comment:
Those computeIfAbsent changes are unrelated and will also result in tab not generating unless tab has elements.

Please revert this one, the other compute if absent below and other changes that are not directly related to the PR.

If you see the need for extra refactoring, please do it in a separate PR.